src/deidentify/address_to_census_tract.py

```diff
@@ -352,8 +352,8 @@ def census_tract_csv_or_excel(response: pd.DataFrame, tracts) -> pd.Series:
     Extract lat/lng from *response* DataFrame and return a pd.Series containing
     the affiliated census tract from the given *tracts* file of polygons.
     """
-    lat = response['response'].apply(lambda x: x['lat'] if 'lat' in x else None)
-    lng = response['response'].apply(lambda x: x['lng'] if 'lng' in x else None)
+    lat = response['response'].apply(lambda x: x.get('lat', None))
+    lng = response['response'].apply(lambda x: x.get('lng', None))
```

Comment:
No problem being explicit here, but worth noting that `.get()` defaults to returning `None` if no second arg is provided. :-)

Reply:
https://docs.python.org/3/library/stdtypes.html#dict.get